generators/server/templates/_build.gradle

```diff
@@ -453,12 +449,23 @@ clean {
     delete "target"
 }
 
+task copyDockerFiles(type: Copy) {
```

Comment:
Cant we have this in the `docker.gradle` file which btw is still present? as I said earlier I was able to make this work in imported file by declaring the below at the top of the `docker.gradle` file
```
// Somehow this plugin works only with a buildscript definition
// Also this plugin seems to be broken on latest Docker version
buildscript {
  repositories {
    jcenter()
    mavenLocal()
  }

  dependencies {
    classpath 'com.bmuschko:gradle-docker-plugin:3.0.6'
  }
}
apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
```